vector-im/element-android
# [Device management] Verify current session (PSG-722)

## Type of change


## Content


Start the verification process when pressing the "Verify" button only for the current Session.
Bonus: Fix missing sending of verification event to other verified sessions in the devices list V1.

## Motivation and context


Closes #7114 

## Screenshots / GIFs




No UI has been changed.

## Tests



- Enable the new device management feature flag
- Sign in on a new device
- Go to Settings -> Security & Privacy -> Show all sessions (V2, WIP)
- Check the current session is displayed as not verified
- Press the "Verify" button
- Check another verified session received the request
- Cancel it
- Go to the session overview screen by pressing the current Session card
- Press the "Verify" button
- Check another verified session received the request
- Do the verification process
- Check the current session is shown as verified in the overview screen
- Press back
- Check the current session is shown as verified in the sessions list screen

## Tested devices

- OS version(s): Android 11

## Checklist
